### PR TITLE
Add jsk_aero_startup

### PR DESCRIPTION
**Pull request by furushchev**
- [x] depends on #919
- [x] validate launch file? (optional)
- [x] review from higashide kun
- [x] cleanup commits

### CODE REVIEW

File: jsk_aero_robot/aero.rosinstall [hunk 1]
```diff
@@ -10,3 +10,7 @@
     local-name: sasabot/win_lin_kinect
     uri: https://github.com/sasabot/win_lin_kinect.git
     version: master
+- git:
+    local-name: longjie/ps4eye
+    uri: https://github.com/longjie/ps4eye.git
```
Comment (k-okada) [inline jsk_aero_robot/aero.rosinstall]: I think this is already released, if not ask @7675t

File: jsk_aero_robot/jsk_aero_startup/package.xml [hunk 2]
```diff
@@ -16,5 +16,10 @@
   <run_depend>aeroeus</run_depend>
   <run_depend>jsk_maps</run_depend>
   <run_depend>jsk_robot_startup</run_depend>
-
+  <run_depend>linux_kinect</run_depend>
```
Comment (k-okada) [inline jsk_aero_robot/jsk_aero_startup/package.xml]: what is this????

Comment (furushchev) [reply jsk_aero_robot/jsk_aero_startup/package.xml]: This package is located here: https://github.com/sasabot/win_lin_kinect
Bridge to bringing kinect functions working on windows into linux.

**Comment by furushchev** [comment]
@taichiH Please comment if you find any.

File: jsk_aero_robot/aero.rosinstall (hunk 1 above)
Comment (k-okada) [reply jsk_aero_robot/aero.rosinstall]: released https://github.com/ros/rosdistro/pull/18439

**Comment by k-okada** [comment]
`linux_kinct` is  not released so added `aero.test.xml` that set 'use_kinect' and 'use_ps4eys',
if we merge https://github.com/ros/ros_comm/pull/1455, we can use
```
 roslaunch_add_file_check(launch/aero.launch use_kinect:=false use_ps4eye:=false ROBOT=sim)
```

**Pull request merged**